serverless.yaml

```diff
@@ -165,12 +165,13 @@ functions:
           - dynamodb:UpdateItem
         Resource:
           - !GetAtt UserTable.Arn
-  
+
   cognitoAutoConfirm:
     handler: packages/api/functions/cognito/auto-confirm-user.handler
 
   cognitoPostAuthN:
     handler: packages/api/functions/cognito/post-authentication.handler
+    iamRoleStatementsName: serverless-cognito-post-auth
```

Comment:
Need to remove this. I know you added it due to the name length limit, but we can't have hardcoded names as it breaks deploying multiple versions into a single account. Maybe find the limit and add a check in the setup script to print an error if the provided app name exceeds the length? Alternatively, we can reduce the resource names for `cognitoAutoConfirm` and `cognitoPostAuthN`.

Reply:
Ok, understood 😄